docs/modules/ROOT/pages/how-tos/metadata/sboms.adoc

```diff
@@ -1 +1,152 @@
-= Inspecting SBOMs
\ No newline at end of file
+= Inspecting SBOMs
+
+== Software bill of materials
+A software bill of materials (SBOM) provides greater transparency for your software supply chain. {ProductName} provides link:https://www.cisa.gov/sites/default/files/2023-04/sbom-types-document-508c.pdf[build SBOMs], which list all the software libraries that a component uses. Those libraries can enable specific functionality or facilitate development. 
+
+You can use an SBOM to better understand the composition of your software. In particular, you can analyze your SBOM to determine if your application has any vulnerable dependencies. Furthermore, to comply with cybersecurity regulations, you might need to provide your customers with the SBOM for your application's components. For example, in the United States, the link:https://www.whitehouse.gov/briefing-room/presidential-actions/2021/05/12/executive-order-on-improving-the-nations-cybersecurity/[Executive Order on Improving the Nation's Cybersecurity] requires vendors to provide SBOMs, and in Europe, the link:https://www.cisa.gov/sites/default/files/2023-09/EU%20Commission%20SBOM%20Work_508c.pdf[Cyber Resilience Act] does the same.
+
+== Viewing an SBOM in the web UI
+
+.Prerequisites
+
+* You must have an application that {ProductName} has successfully built and deployed.
+
+.Procedure
+
+In the console, complete the following steps to download the SBOM for a component:
+
+. Navigate to the *Activity* > *Pipeline runs* tab.
+
+. For the component whose SBOM you want to view, select its most recent pipeline run.
+
+. On the *Pipeline run details* page, select *View SBOM*. 
+.. You can use your web browser to immediately search the SBOM for terms that indicate vulnerabilities in your software supply chain. For example, try searching for "log4j".
+.. You can select *Download* to download the SBOM, or *Expand* to view it full-screen.
+
+[NOTE]
+====
+This SBOM is unsigned. You can still use it for standard business processes, like providing an SBOM to your customers. However, do not use this unsigned SBOM as a legal document. 
```

Comment:
I would just generalize this and not give any specific use cases for it. Maybe something like:

The SBOM is unsigned so there is no way to verify whether it has been tampered with after being pushed to the registry. Additional integrity verification should be performed before distributing the artifact if needed.